docs/faq/proof-of-stake/voting-tickets.md

```diff
@@ -55,7 +55,9 @@ Output          | Description
 
 Missed tickets are tickets that were called to vote, but did not. After a ticket has been missed, it can be revoked[^14763]. The wallet automatically revokes missed tickets it controls (assuming it is unlocked so it can sign the revocation transaction).
 
-All `rebroadcastmissed` does is tell wallet to ask the chain server (`dcrd`) to give it the list of missed tickets again so it can double check if it needs to revoke any. Typically it is not needed unless, for example, it tried to revoke a ticket but could not because the wallet was not unlocked. In other words, if you noticed that revoked != missed, you might need to `rebroadcastmissed`. As said though, there is rarely a need for this since the wallet automatically does it on startup and looks for missed tickets on every block.
+All `rebroadcastmissed` does is tell wallet to ask the chain server (`dcrd`) to give it the list of missed tickets again so it can double check if it needs to revoke any. Typically it is not needed unless, for example, it tried to revoke a ticket but could not because the wallet was not unlocked. In other words, if you noticed that revoked does not equal missed, you might need to `rebroadcastmissed`. As said though, there is rarely a need for this since the wallet automatically does it on startup and looks for missed tickets on every block.
```

Comment:
"not unlocked" == locked. Feel like excessive repetition of "not" in that sentence.